moovingon_ai/README.md

```diff
@@ -0,0 +1,38 @@
+# moovingon.ai
+
+## Overview
+[moovingon.ai](http://moovingon.ai/) revolutionizes the management of 24/7 operations for organizations through advanced AIOps and runbook automation. It serves start-ups, scale-ups, and global enterprises alike by seamlessly integrating into your infrastructure and simplifying the complexities of cloud operations management.
+
+With [moovingon.ai](http://moovingon.ai/), you can achieve unparalleled control and efficiency. It orchestrates monitoring, productivity, and escalation processes across your entire organization. Our technology adapts to your unique tech stack and workflows, providing a centralized hub for comprehensive observability and consistent operations. Experience over 90% automated healing to focus on strategic initiatives and reduce operational costs.
+
+
+## Setup
+
+### Installation
+
+To install the moovingon.ai Datadog integration, log into the moovingon.ai platform, then follow the below steps:
+
+1. Navigate to **Integrations** and select **Add Datadog**.
+
+2. Enter a name for the integration and **Submit**.
+
+3. Proceed to the Datadog OAuth2 screen and click on the **authorize** button.
+
+4. **(Optional)** To receive events from all monitors, edit and click on **install/update the webhook**.
```

Comment:
The authorization process should kick off from Datadog. Please update this to reflect the steps the customer will take within Datadog to when they click authorize

Reply:
I'm not entirely sure what you mean. In step 3 I specified that within datadog he has to click on "authorize". That's pretty much sums it up. There's nothing else he needs to do in Datadog.

Reply:
I assumed it's possible to move the user to moovingon.ai where he will add a datadog integration by using Oauth2. If it's not the case, where do I implement the logic to interact with my app? is it inside the tile?